### PR TITLE
Drop bucket at the begining of the test

### PR DESCRIPTION
**Pull request by romanbracinik**
Jira: CT-XXX
KBC: XXX

na zaklade https://keboolaglobal.slack.com/archives/C055RTFCJ6M/p1719223442824849

Before asking for review make sure that:

## Checklist

- [x] New client method(s) has tests
- [x] Apiary file is updated

## Release

  - [x] I gave the PR a proper label:
    * major (BC break)
    * minor (new feature)
    * patch (backwards compatible fix)
    * no release (just test changes)

### CODE REVIEW

File: tests/Common/TokensTest.php [hunk 1]
```diff
@@ -1325,7 +1327,7 @@ public function testReadOnlyRoleBucketsPermissions(): void
         $this->assertCount($bucketsInitialCount, $buckets);
 
         // create new bucket with master token
-        $newBucketId = $this->_client->createBucket('test', 'in', 'testing');
+        $newBucketId = $this->_client->createBucket($bucketName, 'in', 'testing');
```
Comment (tomasfejfar) [inline tests/Common/TokensTest.php:1330]: tady bys ho mohl zahodit taky, ne?

File: tests/Common/TokensTest.php [hunk 2]
```diff
@@ -928,9 +931,6 @@ public function testAllBucketsTokenPermissions(): void
         foreach ($token['bucketPermissions'] as $bucketId => $permission) {
             $this->assertEquals(self::BUCKET_PERMISSION_MANAGE, $permission);
         }
-
-        $client->getBucket($newBucketId);
```
Comment (tomasfejfar) [inline tests/Common/TokensTest.php:932]: Tohle by se nemělo mazat, to je ten test, ne? Že to nespadne na exceptioně...

File: tests/Common/TokensTest.php [hunk 3]
```diff
@@ -1336,9 +1338,6 @@ public function testReadOnlyRoleBucketsPermissions(): void
         foreach ($token['bucketPermissions'] as $bucketId => $permission) {
             $this->assertEquals('read', $permission);
         }
-
-        $client->getBucket($newBucketId);
```
Comment (tomasfejfar) [inline tests/Common/TokensTest.php:1340]: Tohle by se nemělo mazat, to je ten test, ne? Že to nespadne na exceptioně...

File: tests/Common/TokensTest.php [hunk 4]
```diff
@@ -917,7 +920,7 @@ public function testAllBucketsTokenPermissions(): void
         $this->assertCount($bucketsInitialCount, $buckets);
 
         // create new bucket with master token
-        $newBucketId = $this->_client->createBucket('test', 'in', 'testing');
+        $this->_client->createBucket($bucketName, 'in', 'testing');
```
Comment (tomasfejfar) [inline tests/Common/TokensTest.php:923]: Protože si myslím, že má zůstat get, tak má zůstat i $newBucketId

**Review by tomasfejfar — CHANGES_REQUESTED** [review]

File: tests/Common/TokensTest.php (hunk 2 above)
Comment (romanbracinik) [reply tests/Common/TokensTest.php:932]: No jo vlastne :)

Comment (romanbracinik) [reply tests/Common/TokensTest.php:932]: vratil som to

File: tests/Common/TokensTest.php (hunk 3 above)
Comment (romanbracinik) [reply tests/Common/TokensTest.php:1340]: vratil som to

**Pull request merged**